python/test/runner.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# runner.py
+
+import unittest
+
+import options
+
+from unit import test_contexts
+from unit_distributed import test_contexts_arbmpi
+from unit_distributed import test_contexts_mpi4py
+# add more if needed
+
+test_modules = [\
+    test_contexts,\
+    test_contexts_arbmpi,\
+    test_contexts_mpi4py\
+] # add more if needed
+
+"""
```

Comment:
How about a single string: "returns a suite of all the tests".
The Goal and name are redundant.

Reply:
Done.